src/foam/lib/csv/CSVParser2.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
Document how this is different than the regular one and why I would want to use it.

Reply:
Added documentation and renamed file. Name sound ok?